webapp/src/Controller/Jury/TeamController.php

```diff
@@ -109,8 +109,7 @@ public function indexAction(): Response
         $table_fields = [
             'teamid' => ['title' => 'ID', 'sort' => true,],
             'icpcid' => ['title' => 'ICPC ID', 'sort' => true,],
-            'name' => ['title' => 'teamname', 'sort' => true, 'default_sort' => true],
-            'display_name' => ['title' => 'display name', 'sort' => true, 'default_sort' => true],
+            'effective_name' => ['title' => 'name', 'sort' => true, 'default_sort' => true],
```

Comment:
you are still using `displayName` below, but here `effective_name`. Is that intentional?

Reply:
Where exactly? In the form? Then yes, since effectiveName is the method that checks displayNamw and otherwise name

Reply:
Perhaps I am misunderstanding the distinction between effectiveName and displayName then. What is the difference and why do we need both?